projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -17,56 +11,127 @@ import { CartActions } from '../actions/index';
 
 @Injectable()
 export class CartEffects {
+  // TODO: remove when removing cart store module
+  @Effect()
+  loadCart2$: Observable<CartActions.LoadMultiCart> = this.actions$.pipe(
```

Comment:
Why do we add yet another effects in our public API? Maybe we can move them into separate private API file?